cmd/v/help/doc.txt

```diff
@@ -27,9 +27,6 @@ Options:
 
 For HTML mode:
   -inline-assets  Embeds the contents of the CSS and JS assets into the webpage directly.
-  -open           Launches the browser when the server docs has started.
```

Comment:
cmd/tools/vdoc/utils.v:12:fn open_url(url string) {

Reply:
yeah, you're right it isn't used anywhere